script/glide.sh

```diff
@@ -6,11 +6,6 @@ set -o nounset
 ####
 ### Helper script for glide[-vc] to handle specifics for the Traefik repo.
 ##
-# In particular, the 'integration/' directory contains its own set of
-# glide-managed dependencies which must not have its nested vendor folder
-# stripped. Depending on where the script is called from, it will do the Right
-# Thing.
-#
 
 CWD="$(pwd)"; readonly CWD
```

Comment:
Given that the vendor folder location is now non-ambiguous, we could just `cd` into the right directory relative to the location of this script. That way, the script can be called from just anywhere.